portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,17 +36,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = request.getParameter("text-input");
+    comments.add(text);
+
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println("Your last comment was: ");
+    response.getWriter().println(comments.get(comments.size()-1));
+  }
+
   private String convertToJson() {
+    comments.set(0, "First comment");
```

Comment:
Tip: When the comments get more confusing and the Jsons are tougher to parse, I would recommend importing a Json parser in the future.